src/NuGet.Core/NuGet.Common/Errors/NuGetLogCode.cs

```diff
@@ -92,6 +92,11 @@ public enum NuGetLogCode
         /// </summary>
         NU1009 = 1009,
 
+        /// <summary>
+        /// Central floating versions are not allowed.
+        /// </summary>
+        NU1011 = 1011,
```

Comment:
Can you please create a follow up issue for adding docs for this? 
Or potentially a PR?

Reply:
Added https://github.com/NuGet/Home/issues/9940